bundle/manifests/dell-csm-operator.clusterserviceversion.yaml

```diff
@@ -2929,42 +3479,5 @@ spec:
   provider:
     name: Dell Technologies
     url: https://github.com/dell/csm-operator
-  relatedImages:
-  - image: docker.io/dellemc/dell-csm-operator:v1.4.0
-    name: dell-csm-operator
-  - image: docker.io/dellemc/csi-isilon:v2.9.0
-    name: csi-isilon
-  - image: docker.io/dellemc/csi-powermax:v2.9.0
-    name: csi-powermax
-  - image: docker.io/dellemc/csipowermax-reverseproxy:v2.8.0
-    name: csipowermax-reverseproxy
-  - image: docker.io/dellemc/csi-powerstore:v2.9.0
-    name: csi-powerstore
-  - image: docker.io/dellemc/csi-unity:v2.9.0
-    name: csi-unity
-  - image: docker.io/dellemc/csi-vxflexos:v2.9.0
-    name: csi-vxflexos
-  - image: docker.io/dellemc/sdc:4.5
-    name: sdc
-  - image: docker.io/dellemc/csm-authorization-sidecar:v1.9.0
-    name: karavi-authorization-proxy
-  - image: docker.io/dellemc/dell-csi-replicator:v1.7.0
-    name: dell-csi-replicator
-  - image: docker.io/dellemc/dell-replication-controller:v1.7.0
-    name: dell-replication-controller-manager
-  - image: docker.io/dellemc/csm-topology:v1.7.0
-    name: topology
-  - image: docker.io/otel/opentelemetry-collector:0.42.0
-    name: otel-collector
-  - image: docker.io/dellemc/csm-metrics-powerscale:v1.4.0
-    name: metrics-powerscale
-  - image: docker.io/dellemc/csm-metrics-powermax:v1.2.0
-    name: metrics-powermax
-  - image: docker.io/dellemc/csm-metrics-powerflex:v1.7.0
-    name: metrics-powerflex
-  - image: docker.io/dellemc/podmon:v1.8.0
-    name: podmon-node
-  - image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
-    name: kube-rbac-proxy
   replaces: dell-csm-operator.v1.3.0
```

Comment:
Revert the deleted lines from https://github.com/dell/csm-operator/pull/427/files#diff-684460b60b9e0858ebf0dbe4519848f01d8caef6b6499b8d5fe5b1d5b196913eL2932 to https://github.com/dell/csm-operator/pull/427/files#diff-684460b60b9e0858ebf0dbe4519848f01d8caef6b6499b8d5fe5b1d5b196913eL2968